tajo-core/src/test/java/org/apache/tajo/engine/planner/TestLogicalPlanner.java

```diff
@@ -644,7 +644,7 @@ public final void testJoinWithMultipleJoinQual4() throws IOException, PlanningEx
         , new FieldEval(new Column("default.t.n_nationkey", Type.INT4))
         , new FieldEval(new Column("default.s.s_suppkey", Type.INT4))
     );
-    joinQualMap.put(joinQual, Boolean.FALSE);
+    //joinQualMap.put(joinQual, Boolean.FALSE);
```

Comment:
It would be better if you note that this line must be restored when https://issues.apache.org/jira/browse/TAJO-742 is resolved.
The ```TODO``` annotation will be helpful for the future restoration.